src/pipx/commands/__init__.py

```diff
@@ -23,5 +20,5 @@
     "reinstall_all",
     "list_packages",
     "run_pip",
-    "ensurepath",
+    "ensure_path",
```

Comment:
Before merging, notice this command has been renamed from `ensurepath` to `ensure_path` (see first comment). A decision has to be made:

- Keep `ensurepath` or update docs to reflect `ensure_path`.

Let me know.